modules/appeals_api/config/routes.rb

```diff
@@ -96,27 +94,185 @@
+
+      namespace :forms do
+        resources '200996', only: %i[create show], controller: controller_path do
+          collection do
+            post 'validate'
+          end
+        end
+      end
+
+      namespace :schemas, controller: controller_path do
+        get '200996', action: :schema
+      end
+
+      resources :schemas, only: :show, param: :schema_type, controller: '/appeals_api/schemas/shared_schemas'
+    end
+  end
+
+  namespace :supplemental_claims, path: 'supplemental-claims', defaults: { format: 'json' } do
+    namespace :v0 do
+      controller_path = '/appeals_api/supplemental_claims/v0/supplemental_claims'
+
+      get :healthcheck, to: '/appeals_api/metadata#healthcheck'
+      get :upstream_healthcheck,
+          to: '/appeals_api/metadata#mail_status_upstream_healthcheck',
+          path: 'upstream-healthcheck'
+      get :docs, to: '/appeals_api/docs/v2/docs#sc'
+
+      namespace :forms do
+        resources '200995', only: %i[create show], controller: controller_path do
+          collection do
+            post 'validate'
+          end
+        end
+      end
+
+      resources :evidence_submissions,
+                only: %i[create show],
+                controller: "#{controller_path}/evidence_submissions",
+                path: 'evidence-submissions'
+
+      namespace :schemas, controller: controller_path do
+        get '200995', action: :schema
+      end
+
+      resources :schemas, only: :show, param: :schema_type, controller: '/appeals_api/schemas/shared_schemas'
+    end
+  end
+
+  concern :appealable_issues_routes do |opts|
+    controller_path = '/appeals_api/contestable_issues/v0/contestable_issues'
+    api_name = opts[:deprecated] ? 'contestable-issues' : 'appealable-issues'
+
+    mapper.instance_eval do
+      namespace :appealable_issues, path: api_name, defaults: { format: 'json' } do
+        namespace :v0 do
+          get :appealable_issues,
+              to: "#{controller_path}#index",
+              path: "#{api_name}/:decision_review_type"
+          get :healthcheck, to: '/appeals_api/metadata#healthcheck'
+          get :upstream_healthcheck,
+              to: '/appeals_api/metadata#appeals_status_upstream_healthcheck',
+              path: 'upstream-healthcheck'
+          get :docs, to: '/appeals_api/docs/v2/docs#ci'
+
+          namespace :schemas, controller: controller_path do
+            get 'headers', action: :schema
+          end
+
+          resources :schemas, only: :show, param: :schema_type, controller: '/appeals_api/schemas/shared_schemas'
+        end
+      end
+    end
+  end
+
+  appealable_issues_controller_path = '/appeals_api/contestable_issues/v0/contestable_issues'
 
-  concern :notice_of_disagreements_routes, AppealsApi::SharedRoutes::NoticeOfDisagreements
-  concerns :notice_of_disagreements_routes, deprecated: true
-  concerns :notice_of_disagreements_routes
+  concern :appealable_issues_routes do
```

Comment:
The route concerns in the `route_concerns.rb` file that I deleted in this file are no longer needed, except for this one for contestable issues - I'm keeping this one around so that we can some transition time between using `contestable-issues` and `appealable-issues` - cassowary will need to update their mappings before we can delete the `constestable-issues` versions.